### PR TITLE
Upgrade version support to 3.7 on master branch

### PR DESCRIPTION
**Pull request by mitzkia**
Add support to use docker images with syslog-ng version 3.7.

### CODE REVIEW

File: syslog-ng/Dockerfile [hunk 1]
```diff
@@ -4,12 +4,11 @@ MAINTAINER Andras Mitzki <andras.mitzki@balabit.com>
...
 
-RUN wget -qO - http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng-3.6/Debian_8.0/Release.key | apt-key add -
-RUN echo 'deb http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng-3.6/Debian_8.0 ./' | tee --append /etc/apt/sources.list.d/syslog-ng-obs.list
+RUN wget -qO - http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/Debian_8.0/Release.key | apt-key add -
+RUN echo 'deb http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/Debian_8.0 ./' | tee --append /etc/apt/sources.list.d/syslog-ng-obs.list
 
 RUN apt-get update -qq && apt-get install -y \
-    syslog-ng \
-    syslog-ng-mod*
+    syslog-ng
```
Comment (ihrwein) [inline syslog-ng/Dockerfile:11]: In `3.6` we installed `syslog-ng` and `syslog-ng-mod*`. Don't we need to this with `3.7`?

**Comment by ihrwein** [comment]
I've placed my review notes.

File: syslog-ng/Dockerfile (hunk 1 above)
Comment (mitzkia) [inline syslog-ng/Dockerfile:11]: In issue https://github.com/balabit/syslog-ng/issues/677 we verified that "syslog-ng" package installs all module packages for syslog-ng, so we do not need to install "syslog-ng-mod*".

**Comment by ihrwein** [comment]
:+1: thanks :smile:

**Pull request merged**